data/en/lseurocurrencyformat.json

```diff
@@ -1,13 +1,14 @@
 {
 	"name":"lsEuroCurrencyFormat",
 	"type":"function",
-	"syntax":"lsEuroCurrencyFormat(currency, type)",
+	"syntax":"lsEuroCurrencyFormat(currency [, type, locale])",
```

Comment:
It says that `type` is required so it should not be in square brackets here right?

Reply:
I'm not shure if type is required.
lsEuroCurrencyFormat(5) works with "currency" only

Reply:
Yes, according to the adobe docs `local` is the default type, so we should just update the `required` and `default` on the param below. Can you take care of that?

Reply:
Of course!
I do some updates and commit them.